build/cpp/func/src/func_test.cc

```diff
@@ -54,23 +52,6 @@ TEST(Func, GetEvent) {
   EXPECT_EQ(reply.followers()[0], "username_0");
 }
 
-// test invalid event
```

Comment:
Invalid parents are no longer relevant in warble functions as the warble ids are randomly created at the commandline.